### ISSUE TITLE
Some cool filters are skipped because they have no UI

### ISSUE THREAD

**Issue by boogiefromzk** [ivyl/obs-lv2#3]
I think that if the filter has no UI you can simply show a message that there's no UI on filter. 
I had to remove the part with /* filter out plugins without supported UI */ to make some cool filters work.

**Comment by ivyl** [comment]
Fair point. I am almost done with implementation of the generic UI mentioned in #1. I am happy that you managed to make it work for you.

**Comment by michaelcadilhac** [comment]
@ivyl Don't hesitate to make the branch public if you want some beta testers :-)  Cheers!
